commands/publish_dafsa.py

```diff
@@ -0,0 +1,100 @@
+import requests
+from requests.exceptions import HTTPError
+import tempfile
+import subprocess
+from kinto_http import Client, KintoException
+import sys
+
+COMMIT_HASH_URL = "https://api.github.com/repos/publicsuffix/list/commits/master"
+
+LIST_URL = (
+    "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat"
+)
+
+MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
+PREPARE_TLDS_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py"
+
+
+SERVER = "https://kinto.dev.mozaws.net/v1"
+CREDENTIALS = ("arpit73", "s3cr3t")  # (username, password)
```

Comment:
See other lambdas functions, we take those from an `event` param and/or from `os.getenv()`

Reply:
I just replaced these with environment variables as I has planned previously. What do you mean by lambda functions and `event` in this case?

Reply:
See how other commands do in this repo...

The Amazon Lambda sends us an `event` (a dict) with some configuration values. When missing from event, they should be taken from `os.getenv()` with a default value :)

Reply:
Will look into this!

Reply:
looking at the scripts, it seems like we can get all these global values from the `event` object or as an environment object.
```python
COMMIT_HASH_URL = "https://api.github.com/repos/publicsuffix/list/commits?path=public_suffix_list.dat"
LIST_URL = "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat"

MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
PREPARE_TLDS_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py"

SERVER = "https://kinto.dev.mozaws.net/v1"
CREDENTIALS = (os.getenv("USERNAME"), os.getenv("PASSWORD"))  # (username, password)

BUCKET_ID = "firefox-core-network-dns"
COLLECTION_ID = "public-suffix-list"
RECORD_ID = "latest-commit-hash"
```

Since the current version uses these as globals, I will have to modify each function to get their respective values as parameters instead.

Reply:
This would be easier to write if knew what the `event` object looks like and what values we are planning to send using this, shall I assume the `event` to be like this?

```python
event = {
    "COMMIT_HASH_URL": "https://api.github.com/repos/publicsuffix/list/commits?path:public_suffix_list.dat",
    "LIST_URL": "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat",
    
    "MAKE_DAFSA_PY": "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py",
    "PREPARE_TLDS_PY": "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py",
    
    "SERVER": "https://kinto.dev.mozaws.net/v1",
    "AUTH": ("USERNAME", "PASSWORD"),
    
    "BUCKET_ID": "firefox-core-network-dns",
    "COLLECTION_ID": "public-suffix-list",
    "RECORD_ID": "latest-commit-hash",
}
```

Reply:
you don't have to put everything configurable. Start with SERVER and AUTH